core/build.zig.zon

```diff
@@ -1,15 +0,0 @@
-.{
-    .name = "core",
```

Comment:
Why did you remove this package? Do you understand that you've broken the packaging system?

Reply:
My intention was to include core (microzig, start and cpus) directly into the root package (didn't update each package's files list, yet).

Reply:
I added it back